arrow/util.py

```diff
@@ -12,12 +12,30 @@ def is_timestamp(value):
     if isinstance(value, bool):
         return False
     try:
-        datetime.fromtimestamp(value)
+        datetime.datetime.fromtimestamp(value)
         return True
     except TypeError:
         return False
 
 
+# Credit to https://stackoverflow.com/a/1700069
+def iso_to_gregorian(iso_year, iso_week, iso_day):
+    """Converts an iso weekday tuple into a datetime object."""
+
+    if iso_week == 0 or iso_day == 0:
```

Comment:
We should probably split this into two conditionals so we can make the error messages clearer.